### PR TITLE
feat: less to component token for Typography

### PR DESCRIPTION
**Pull request by jrr997**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
ref https://github.com/ant-design/ant-design/issues/41884

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | less to component token for Typography |
| 🇨🇳 Chinese | 排版组件less变量迁移 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 4c574de</samp>

This pull request updates the documentation for the Modal component to show how to migrate from less variables to component tokens for typography. It covers both the English and Chinese versions of `docs/react/migrate-less-variables.md`.

### <samp>🤖 Generated by Copilot at 4c574de</samp>

* Add tables of less variables and component tokens for typography of Modal component in English and Chinese documentation ([link](https://github.com/ant-design/ant-design/pull/42044/files?diff=unified&w=0#diff-999566baa64628c49813370e7cb13c3c8f5e0c99ef8fe9cec7c5052985c10d9aR39-R47), [link](https://github.com/ant-design/ant-design/pull/42044/files?diff=unified&w=0#diff-dff3e0a0a5fae848621e55da4e2adaf0ec6c57d5e29d3f601799c8898c280739R39-R47))

### CODE REVIEW

File: docs/react/migrate-less-variables.zh-CN.md [hunk 1]
```diff
@@ -36,3 +36,12 @@ title: less 变量迁移 Component Token
...
 | `@modal-border-radius` | `borderRadiusLG` | - |
+
+## Typography 排版
+
+<!-- prettier-ignore -->
+| less 变量 | Component Token | 备注 |
+| --- | --- | --- |
+| `@typography-title-font-weight` | - | 由于样式变化已废弃 |
+| `@typography-title-margin-top` | `sizeMarginHeadingVerticalStart` | - |
+| `@typography-title-margin-bottom` | `sizeMarginHeadingVerticalEnd` | - |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:47]: 这个 token 加到 Component Token 里，然后用后面的 token 初始化即可。margin 这类 token 很容易对其他地方的样式产生影响，所以不推荐直接用 global token

File: docs/react/migrate-less-variables.zh-CN.md [hunk 2]
```diff
@@ -36,3 +36,12 @@ title: less 变量迁移 Component Token
 | `@modal-confirm-body-padding` | - | 由于样式变化已废弃 |
 | `@modal-confirm-title-font-size` | `modalHeaderTitleFontSize` | - |
 | `@modal-border-radius` | `borderRadiusLG` | - |
+
+## Typography 排版
+
+<!-- prettier-ignore -->
+| less 变量 | Component Token | 备注 |
+| --- | --- | --- |
+| `@typography-title-font-weight` | - | 由于样式变化已废弃 |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:45]:
```suggestion
| `@typography-title-font-weight` | `fontWeightStrong` | - |
```

File: docs/react/migrate-less-variables.zh-CN.md (hunk 1 above)
Comment (jrr997) [reply docs/react/migrate-less-variables.zh-CN.md:47]: `sizeMarginHeadingVerticalStart` 和 `sizeMarginHeadingVerticalEnd` 是写在 Typography 组件的 component token 里面的，不是 global token。搜了一下全局只有 Typography 有这个 token。大佬确认一下。

Comment (MadCcc) [reply docs/react/migrate-less-variables.zh-CN.md:222]: 那应该改名叫 `titleMarginTop` 和 `titleMarginBottom` 了

**Comment by MadCcc** [comment]
应该 merge feature 的

**Comment by MadCcc** [comment]
这个 PR 好像 reopen 不了了，重开一个吧

**Comment by jrr997** [comment]
手欠点了同步master，我重开一个。

**Comment by yoyo837** [comment]
> 手欠点了同步master，我重开一个。

这可能就是当初禁用那个功能的原因了 @afc163

**Comment by MadCcc** [comment]
上游分支是 feature，应该不会同步 master 才对吧

**Comment by jrr997** [comment]
重开了 https://github.com/ant-design/ant-design/pull/42442

**Comment by jrr997** [comment]
> 上游分支是 feature，应该不会同步 master 才对吧

我在自己fork的antd仓库点了 Sync Fork。

**Pull request closed without merging**